src/compat/object/set.spec.ts

```diff
@@ -10,7 +10,6 @@ describe('set', () => {
       a: number;
     }
     const result = set<Test>({}, 'a', 1);
-    result.a;
```

Comment:
wasn't sure why it was there, and eslint threw an error
is it necessary?

Reply:
Seems that it was from lodash's test code, I don't think it's necessary.